examples/view_clearcontrol_dataset.py

```diff
@@ -0,0 +1,39 @@
+import sys
+import tempfile
+import time
+
+from iohub.clearcontrol import (
+    ClearControlFOV,
+    create_mock_clear_control_dataset,
+)
+
+
+def main():
+    """Example of opening a Clear Control dataset and viewing with napari.
+    """
+    import napari
+
+    if len(sys.argv) < 2:
```

Comment:
Can you document what the arguments should be?

Reply:
Added usage example